apps/condo/domains/contact/components/ContactsEditor/NewContactFields.tsx

```diff
@@ -1,13 +1,15 @@
 import { MinusCircleOutlined } from '@ant-design/icons'
 import styled from '@emotion/styled'
-import { AutoComplete, Col, Form, Radio } from 'antd'
+import { AutoComplete, Col, Form, InputProps, Radio, Row, RowProps } from 'antd'
```

Comment:
We have radio in UI-kit, may be we can change it here?

Reply:
changed